### PR TITLE
Feat/update destination

### PR DESCRIPTION
**Pull request by derjubu**
Implement update function for Destinations.
Now a user can update the location and the dates for each destination.

### CODE REVIEW

**Review by manusmd — APPROVED** [review]
lgtm🚀

File: src/app/pages/UpdateDestination/UpdateDestination.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { useNavigate } from 'react-router';
+
+export default function UpdateDestination(): JSX.Element {
+  const currentId = window.localStorage
+    .getItem('UpdateId')
+    ?.replaceAll('"', '');
+
+  const currentDestination: any[] = JSON.parse(
+    window.localStorage.getItem('destination') as string
+  );
```
Comment (marwinburesch) [inline src/app/pages/UpdateDestination/UpdateDestination.tsx:19]: I think you still have that old `useLocalStorage` hook lying around somewhere in the attic, I suggest using it :)

File: src/server.ts [hunk 2]
```diff
@@ -24,11 +23,28 @@ app.post('/api/location/', async (request, response) => {
...
 });
 
+app.get('/api/location/:id', async (request, response) => {
+  const { id } = request.params;
+  const findLocation = await getItinerary()
+    .find({
+      _id: new ObjectId(id as string),
+    })
+    .toArray();
+  console.log(findLocation);
```
Comment (marwinburesch) [inline src/server.ts:38]: 🧐 Maybe some more context?

File: src/server.ts [hunk 3]
```diff
@@ -24,11 +23,28 @@ app.post('/api/location/', async (request, response) => {
...
+  if (findLocation) {
+    console.log('Done');
+    response.status(200).send(findLocation);
+  } else {
+    console.log('Error');
```
Comment (marwinburesch) [inline src/server.ts:43]: If you like to log these errors, you should use `console.error()`

File: src/server.ts [hunk 4]
```diff
@@ -24,11 +23,28 @@ app.post('/api/location/', async (request, response) => {
...
+app.get('/api/location/:id', async (request, response) => {
+  const { id } = request.params;
+  const findLocation = await getItinerary()
+    .find({
+      _id: new ObjectId(id as string),
+    })
+    .toArray();
+  console.log(findLocation);
+  if (findLocation) {
+    console.log('Done');
```
Comment (marwinburesch) [inline src/server.ts:40]: Done with what?

**Review by marwinburesch — COMMENTED** [review]
Nice

**Pull request merged**